### PR TITLE
Add friday afternoon check

### PR DESCRIPTION
**Pull request by phaseinducer**

### CODE REVIEW

**Comment by anaelChardan** [comment]
:joy:

File: src/Release/Command/TagCommand.php [hunk 1]
```diff
@@ -177,4 +182,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $io->text(file_get_contents(getcwd(). '/src/Release/resources/done.txt'));
     }
+
+    private function fridayAfternoonCheck(StyleInterface $io): bool
+    {
+        if ('5' === date('N') && '14' < date('G')) {
+            return $io->confirm("It's friday afternoon, are you sure you want to tag now?", false);
```
Comment (AymericPlanche) [inline src/Release/Command/TagCommand.php]:
```suggestion
            return $io->confirm("It's friday afternoon, are you sure you want to tag now? If so, warn your teammates with your speakers on: https://www.youtube.com/watch?v=kfVsfOSbJY0", false);
```

**Comment by tamarasaurus** [comment]
Needs a number of beers check

File: src/Release/Command/TagCommand.php (hunk 1 above)
Comment (phaseinducer) [reply src/Release/Command/TagCommand.php]: :redcard:

Comment (pierallard) [reply src/Release/Command/TagCommand.php]: Report obscene

**Comment by anaelChardan** [comment]
10 reviews while we sometimes have problem to get 2 :joy:

**Pull request merged**